gef.py

```diff
@@ -206,15 +210,15 @@ def update_gef(argv):
 __gef_int_stream_buffer__              = None
 __gef_redirect_output_fd__             = None
 
+HOME_DIR = os.getenv("HOME")
 DEFAULT_PAGE_ALIGN_SHIFT               = 12
 DEFAULT_PAGE_SIZE                      = 1 << DEFAULT_PAGE_ALIGN_SHIFT
-GEF_RC                                 = os.path.join(os.getenv("HOME"), ".gef.rc")
-GEF_TEMP_DIR                           = os.path.join(tempfile.gettempdir(), "gef")
+GEF_RC                                 = os.path.join(HOME_DIR, ".gef.rc")
+GEF_TEMP_DIR                           = os.path.join(HOME_DIR, ".cache", "gef")
```

Comment:
I'd much prefer something like:
```python
os.path.join(tempfile.gettempdir(), os.getenv("USER"), "gef")
```

`gettempdir()` has the advantage of being deleted every reboot. Why use `~/.cache` in particular?

Reply:
What if an user on the same shared server squat `/tmp/<user>` of other users?

Reply:
You will have the sticky bit to protect your stuff:
```
hugsy@ubuntu-dev ~ $ mkdir /tmp/hugsy
hugsy@ubuntu-dev ~ $ touch /tmp/hugsy/foo
hugsy@ubuntu-dev ~ $ sudo -u user touch /tmp/hugsy/bar
touch: cannot touch '/tmp/hugsy/bar': Permission denied
```

Reply:
Placing stuff in `~/.cache` avoid that problems. 
Most browsers store their cache, and some software store video/picture thumbnails there.